sql/updates/latest-dev.sql

```diff
@@ -161,4 +161,19 @@ DROP FUNCTION IF EXISTS _timescaledb_functions.get_chunk_colstats;
 DROP FUNCTION IF EXISTS _timescaledb_internal.get_chunk_relstats;
 DROP FUNCTION IF EXISTS _timescaledb_internal.get_chunk_colstats;
 
-
+-- In older TSDB versions, we disabled autovacuum for compressed chunks
+-- to keep the statistics. However, this restriction was removed in
+-- #5118 but no migration was performed to remove the custom
+-- autovacuum setting for existing chunks.
+DO $$
+DECLARE
+  chunk regclass;
+BEGIN
+  FOR chunk IN
+  SELECT format('%I.%I', schema_name, table_name)::regclass
+    FROM _timescaledb_catalog.chunk WHERE compressed_chunk_id IS NOT NULL
```

Comment:
hmm i would remove the IS NOT NULL check, otherwise i think it might not get properly reset in certain situation
consider the following:
- chunk was compressed with vacuum disabled
- upgrade to version that removed the vacuum disable code
- chunk decompressed
the chunk might now still be flagged as vacuum disabled

Reply:
maybe the check should be for chunks of public hypertables, but i dont see problem removing it from all